nemo/core/classes/modelPT.py

```diff
@@ -92,11 +133,16 @@ def save_to(self, save_path: str):
         with tempfile.TemporaryDirectory() as tmpdir:
             config_yaml = path.join(tmpdir, _MODEL_CONFIG_YAML)
             model_weights = path.join(tmpdir, _MODEL_WEIGHTS)
+            if hasattr(self, 'artifacts') and self.artifacts is not None:
+                for (conf_path, src) in self.artifacts:
+                    try:
+                        if os.path.exists(src):
+                            shutil.copy2(src, tmpdir)
+                    except:
+                        logging.error(f"Could not copy artifact {src} used in {conf_path}")
```

Comment:
PEP8, lets use `except Exception:` to catch the general exceptions.

Reply:
changed, but why is "except Exception" better than just "except" ?

Reply:
Its not better but PEP8 has weird rules.